### PR TITLE
RESTWS-782: upgrade org.apache.tomcat from 6.0.18 to 6.0.53

### PR DESCRIPTION
**Pull request by achilep**
## Issue I worked on

see https://issues.openmrs.org/browse/RESTWS-782

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by achilep** [comment]
Screenshot from 2020-06-27 12-13-58

File: omod-1.11/pom.xml [hunk 1]
```diff
@@ -110,7 +110,7 @@
 		<dependency>
 		      <groupId>org.apache.tomcat</groupId>
 		      <artifactId>jasper</artifactId>
-		      <version>6.0.18</version>
+		      <version>6.0.53</version>
```
Comment (HerbertYiga) [inline omod-1.11/pom.xml:113]: could this be the current version of this?

Comment (achilep) [reply omod-1.11/pom.xml:113]: @HerbertYiga [here](https://search.maven.org/artifact/org.apache.tomcat/jasper)  we might see that 6.0.53 i the last version and it    was release  in 2017

**Comment by sherrif10** [comment]
Did you intend not to add the dependency in the parent pom , havent seen it, may be you can direct to me that change, otherwise thanks for great work

**Comment by achilep** [comment]
> Did you intend not to add the dependency in the parent pom , havent seen it, may be you can direct to me that change, otherwise thanks for great work

no , i will not add a dependencies on the parent pom

**Comment by gitcliff** [comment]
@dkayiwa @ibacher @mozzy11 

Otherwise LGTM

**Comment by sherrif10** [comment]
Sure this LGTM  too cc @dkayiwa

**Comment by dkayiwa** [comment]
Can we use a variable instead of duplicating the version number?

**Comment by achilep** [comment]
variable ? @dkayiwa  @sherrif10 @gitcliff i don't understand , any explanation will be welcome

**Comment by gitcliff** [comment]
@achilep make use of the properties element ie   ```
  <properties>
        <openMRSVersion>2.3.1</openMRSVersion>
    </properties>

```and then just use `openMRSVersion` in the rest of the ther places where the version number is

**Comment by achilep** [comment]
thanks @gitcliff

**Comment by achilep** [comment]
done

**Comment by dkayiwa** [comment]
@achilep is this the latest version as of today?

**Comment by achilep** [comment]
yes @dkayiwa

**Comment by dkayiwa** [comment]
Do you mind sharing the link to it?

**Comment by achilep** [comment]
this link https://issues.openmrs.org/browse/RESTWS-782 Or?

**Comment by dkayiwa** [comment]
I mean link showing it is the latest version.

**Comment by achilep** [comment]
this one ; https://github.com/openmrs/openmrs-module-webservices.rest/pull/439#event-3608355940 ?

**Comment by dkayiwa** [comment]
What i mean is, can you share a link which helped you know that this is the latest released version?

**Comment by achilep** [comment]
is from the image below , the line after the commit message .
Screenshot from 2020-07-31 15-50-34

**Comment by dkayiwa** [comment]
Ok let me ask you this way. If one asked you that what is the latest released version of this dependency, where do you look to answer that?

**Comment by achilep** [comment]
thanks , sorry , this is the link : https://search.maven.org/artifact/org.apache.tomcat/jasper/6.0.53/jar

**Pull request merged**